src/tutorial/runtime/runtime-configuration.rst

```diff
@@ -327,7 +327,8 @@ Start, Stop, Restart
       Add three more ``get_observations`` tasks for each of the remaining
       weather stations.
 
-      You will need the codes for the other three weather stations, which are:
+      You will need the API key and the site codes for the other three
+      weather stations, which are:
```

Comment:
What was wrong with this instruction?

Reply:
> What was wrong with this instruction? I don't think this has changed since Cylc 7.

You need to set both `SITE_ID` _and_ `API_KEY`. IMO "codes" is not sufficiently clear that your are setting the same `API_KEY` and a different `SITE_ID`.  Although the context is such I'm happy to revert and ignore.

> This change makes it sound like they need a different API key for each site.

Agreed, modifying in line with this.

Reply:
Or leave as is and add "replacing the API key as before".